Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -21,7 +21,9 @@
     '2023D2Timing',
     '2023D2TimingPU',
     '2023D3Timing',
-    '2023D3TimingPU'
+    '2023D3TimingPU',
+    '2023D4',
+    '2023D4PU',
```

Comment:
I don't see any obvious typos, but the rearrangement of keys will change the timing WF numbers (fine with me since they just entered the build yesterday)